packages/block-library/src/table/deprecated.js

```diff
@@ -8,18 +8,111 @@ import classnames from 'classnames';
  */
 import { RichText, getColorClassName } from '@wordpress/block-editor';
 
-/**
- * Internal dependencies
- */
-import metadata from './block.json';
-
 const supports = {
 	align: true,
 };
 
 const deprecated = [
 	{
-		attributes: metadata.attributes,
```

Comment:
Would it be possible to filter out the new attributes, instead of duplicating the whole object?

Reply:
I'm not a fan of that approach as it seems quite error prone for someone adding a new deprecation. Once these attributes are defined for a deprecation ideally they won't ever change.